gnome-image-installer/pages/install/gis-scribe.c

```diff
@@ -407,13 +428,15 @@ gis_scribe_init (GisScribe *self)
 GisScribe *
 gis_scribe_new (GFile       *image,
                 guint64      image_size,
+                guint64      compressed_size,
                 GFile       *signature,
                 const gchar *drive_path,
                 gint         drive_fd,
                 gboolean     convert_to_mbr)
 {
   g_return_val_if_fail (G_IS_FILE (image), NULL);
   g_return_val_if_fail (image_size > 0, NULL);
+  g_return_val_if_fail (compressed_size > 0, NULL);
```

Comment:
`0` is allowed as a valid value in the `GParamSpec`. Which one is correct?

Reply:
A GPT disk image must be at least 1 + (2 * 33) 512-byte sectors long (MBR, two copies of (GPT header plus at least 16384 bytes for partition entries)). You could imagine a compression format that compresses a degenerate GPT disk image to one byte. I'll add these minima.